tensorflow_probability/python/sts/smooth_seasonal.py

```diff
@@ -44,7 +159,43 @@ def __init__(self,
                validate_args=False,
                allow_nan_stats=True,
                name=None):
-    """Build a smooth seasonal state space model."""
+    """Build a smooth seasonal state space model.
+
+    Args:
+      num_timesteps: Scalar `int` `Tensor` number of timesteps to model
+        with this distribution.
+      period: positive scalar `float` giving the number of timesteps required
+        for the longest cyclic effect to repeat.
+      frequency_multipliers: one-dimensional `float` `Tensor` giving the number
+        of times each cyclic component repeats per period. Two latent states are
+        required for each element. A 'full' smooth seasonal model is given by
+        `frequency_multipliers = [1, 2, ..., floor(period / 2)]` but it is often
+        reasonable to drop some of these.
```

Comment:
Just a few nits on this:
- I worry that a naive user might not understand that this argument itself *defines* the list of cyclic components; one could read this as referring to a set of 'cyclic components' that were originally defined somewhere else (and then it's confusing to understand where).
- apologies if we've been inconsistent on this elsewhere, but for linear Gaussian models we prefer latent 'dimensions' to latent 'states' to describe the length of the state vector (since the latent state is a continuous-valued random variable with uncountably infinitely many values, as opposed to say an HMM where the latent states are discrete). 
- Maybe say something about *why* it would be reasonable to drop frequency components? 

Here's a stab at a modified version, feel free to adopt whichever parts you like:

One-dimensional `float` `Tensor` listing the frequencies (cyclic components) included in the model, as multipliers of the base frequency `1./period`. Each component is specified by the number of times it repeats per period, and adds two latent dimensions to the model. A smooth seasonal model that can represent any periodic function is given by `frequency_multipliers = [1, 2, ..., floor(period / 2)]`. However, it is often desirable to enforce a smoothness assumption (and reduce the computational burden) by dropping some of the higher frequencies.

Reply:
Cool! I went with what you suggested, it's definitely better than what I had written, I just wasn't 100% sure that the base frequency is `1. / period` though, is the base/fundamental frequency not `2. * pi / period`?

I've updated this in 1fd6d9a

Reply:
Ah, afaik the `2*pi` is just there as a rescaling constant to correct for the fact that trig functions naturally have a period of 2*pi, because they're modeling a journey around the circumference of the unit circle. If you want to build a sin function with period p, you have to write it as `sin(2*pi / p * x)`, etc. Frequency as I learned it is basically just *defined* as the reciprocal of period: a sound wave with frequency 440Hz has a period of 1/440s, etc. (and wikipedia seems to agree: https://en.wikipedia.org/wiki/Frequency). So I think `1./period` is correct here.